index.html

```diff
@@ -2445,7 +2450,13 @@ <h3>
 
             <p>
 The <a href="https://www.w3.org/TR/json-ld11/#the-context">JSON-LD
-specification</a> defines values that are valid for this property.
+specification</a> defines values that are valid for this property. This
+property contains representation-specific syntax and therefore could
+be present in the abstract data model to aid in lossless conversion. If
+the property is present in the abstract data model, it MUST be used
+during production unless either an alternative <code>@context</code> value is
+explicitly provided to the producer or if the value from the abstract data
+model is not valid according to the consumption rules.
```

Comment:
I think this is to address the possible `"@context": true` scenario. If so, it may be clearer to say: 
```suggestion
explicitly provided to the producer or if the value from the abstract data
model is not valid according to the JSON-LD Representation's consumption rules.
```